tests/integration_tests/test_connection_configuration_integration.py

```diff
@@ -27,6 +30,7 @@ class TestPostgresConnectionPutSecretsAPI:
     def url(self, oauth_client: ClientDetail, policy, connection_config) -> str:
```

Comment:
This `TestPostgresConnectionPutSecretsAPI` could also probably have the marks on the class instead of individual functions -